### ISSUE TITLE
 [Masthead (All components under Masthead)]: QA: After expanding the mentioned element, on next tab keyboard focus is no where visible.

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8844]
### Description

Page Element: 
Product & Solutions
Consulting & Services
Learn & Support

Issue:
After expanding the mentioned element, on next tab keyboard focus is no where visible. On the next tab, focus is observed on the first element in the opened menu.

Expected:
After expanding the mentioned element, on next tab keyboard focus should be observed on the first element in the opened menu.

### Component(s) impacted

Masthead:
Default
With custom type ahead
Search open on load
With L1
With alternate logo and tool tip

### Browser

Firefox

### Carbon for IBM.com version

v1.34.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components/?path=/story/components-masthead--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
